.golangci.yml

```diff
@@ -215,7 +215,7 @@ linters:
     - durationcheck # checks for two durations multiplied together
     - errname # checks that sentinel errors are prefixed with the Err and error types are suffixed with the Error
     - errorlint # finds code that will cause problems with the error wrapping scheme introduced in Go 1.13
-    - execinquery # checks query string in Query function which reads your Go src files and warning it finds
+#    - execinquery # checks query string in Query function which reads your Go src files and warning it finds
```

Comment:
а где создало проблемы? Лучше бы конечно локально в нужном месте написать // nolint и причину

Reply:
В новой версии golangci-lint он deprecated и выдает варнинг:
```bash
golangci-lint run
WARN [config_reader] The configuration option `linters.errcheck.ignore` is deprecated, please use `linters.errcheck.exclude-functions`. 
WARN The linter 'execinquery' is deprecated (since v1.58.0) due to: The repository of the linter has been archived by the owner.  
WARN The linter 'gomnd' is deprecated (since v1.58.0) due to: The linter has been renamed. Replaced by mnd. 
```

Reply:
Активировал обратно, пока замены все равно нет.

Reply:
Интересно, не замечал, ну да, в новой версии скорее всего обновят просто